postgresql/client.go

```diff
@@ -558,8 +553,12 @@ func (c *Client) buildCommand(q *prompb.Query) (string, error) {
 	return c.buildQuery(q)
 }
 
-func escapeValue(str string) string {
-	return strings.Replace(str, `'`, `\'`, -1)
+func escapePgString(str string) string {
+	return strings.Replace(strings.Replace(str, `\`, `\\`, -1), `'`, `\'`, -1)
```

Comment:
I'd rather go for using https://golang.org/pkg/strings/#Replacer, it will be more efficient and nicer to read.